schema/metaschema/readme.md

```diff
@@ -0,0 +1,19 @@
+# OSCAL Metaschema directory
```

Comment:
@wendellpiez you may also want to include instructions for executing the `run-metaschema.sh` script in this readme.